packages/xo-web/src/xo-app/settings/plugins/index.js

```diff
@@ -134,6 +134,9 @@ class Plugin extends Component {
               />
               <span className='text-primary'>{` ${props.name} `}</span>
               <span>{`(v${props.version}) `}</span>
+              {props.description !== undefined && props.description !== '' && (
```

Comment:
Why would the description be `''`? Does it make sense?

Reply:
It means that no description for the plugin.